client/src/templates/BaseTemplate/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { useHistory } from "react-router-dom";
+import { Div, Row, Col } from "atomize";
+import { Logo } from "@/components";
+import "./style.scss";
+import githubLogo from "../../assets/imgs/GitHub-Mark/PNG/GitHub-Mark-120px-plus.png";
```

Comment:
If we are only using one image, then just keep one and remove the others. (Might be nice to keep both light/dark version though)

Reply:
I agree yeah